presto-analyzer/src/main/java/com/facebook/presto/sql/analyzer/SemanticErrorCode.java

```diff
@@ -119,4 +120,9 @@ public enum SemanticErrorCode
     TOO_MANY_GROUPING_SETS,
 
     INVALID_OFFSET_ROW_COUNT,
+    INVALID_COPARTITIONING,
```

Comment:
I am wondering if we should have separate error codes for table functions, like TABLE_FUNCTION_.*